bin/common.bash

```diff
@@ -107,7 +107,7 @@ ask_abort() {
 check_tools() {
   local req
 
-  req="${root_path}/roles/get-requirements.yaml"
+  req="$("${scripts_path}/requirements/parse.py")"
```

Comment:
This pulls in Python in basically every `ck8s` invocation, do we want this? I don't believe we use Python anywhere else in `ck8s` scripts, or outside of Ansible? Python is not exactly known for its amazing startup time, although it seems to be vastly overshadowed by `sops --version` doing some http call.

Reply:
Implementing a naive parser in bash should be fairly straight forward but I'd just stick with this for now. We're planning on rewriting a lot of this in Go soon.